src/Sylius/Bundle/ShopBundle/templates/product/show/page/info/overview/accordion/attributes/content.html.twig

```diff
@@ -5,11 +5,18 @@
     {% block accordion_item_content %}
         <table class="table table-lg table-list" {{ sylius_test_html_attribute('product-attributes') }}>
             <tbody>
-            {% for attribute in product.getAttributesByLocale(configuration.request.locale, configuration.request.defaultLocale, sylius_base_locale)|sort_by('attribute.position') %}
+            {% for attribute_value in product.getAttributesByLocale(configuration.request.locale, configuration.request.defaultLocale, sylius_base_locale)|sort_by('attribute.position') %}
                 <tr>
-                    <th scope="row" {{ sylius_test_html_attribute('product-attribute-name', attribute.name) }}>{{ attribute.name }}</th>
-                    <td {{ sylius_test_html_attribute('product-attribute-value', attribute.name) }}>
-                        {{ attribute.value }}
+                    <th scope="row" {{ sylius_test_html_attribute('product-attribute-name', attribute_value.name) }}>{{ attribute_value.name }}</th>
+                    <td {{ sylius_test_html_attribute('product-attribute-value', attribute_value.name) }}>
+                        {% include [
+                            '@SyliusAttribute/Types/'~attribute_value.type~'.html.twig',
```

Comment:
I wonder if we moved this into the shop, like in the admin extending the defaults from the attributes, it would be easier to overwrite them or add a new type, wdyt?